orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -89,22 +94,20 @@ class TaskController {
 
     def orchestrations = executionRepository
```

Comment:
another possible situation where we were fetching tasks in parallel?

Reply:
Also another case of getting carried away.